lib/web_ui/lib/ui_web/src/ui_web/flutter_views_proxy.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'dart:js_interop';
+import 'package:ui/src/engine.dart';
+
+/// The proxy instance for the internal `viewManager` of this app.
+final FlutterViewManagerProxy viewsProxy = FlutterViewManagerProxy();
+
+/// Proxy class for the internal [FlutterViewManager].
+class FlutterViewManagerProxy {
```

Comment:
I think we should find a way to not qualify the class name with "proxy". That adds extra mental load when thinking about what this class does. Internal classes and internal classes. They should not affect the public API surface of `dart:ui_web`. How about we just call it `FlutterViews`?

The dartdoc should also avoid describing the class in terms of an internal details. I propose this:

```
/// Provides additional utility methods for working with `FlutterView` objects.
```

(this comment applies throughout the file)

Reply:
Yes, I can remove the word `Proxy` from the instance at least, and make it so you do:

```dart
ui_web.flutterViews.getInitialData(int viewId);
```

I'm trying to make it somewhat different from `platformViewManager` that we already have.

Reply:
I kept the proxy name in the class, BUT called the instance `views` only. Users should call this like:

```dart
ui_web.views.getInitialData(int viewId);
```